packages/web-components/src/components/scroll-animations/scroll-animations.ts

```diff
@@ -74,13 +77,28 @@ class DDSFadeInOut extends StableSelectorMixin(LitElement) {
    */
   private _resizeObserver: any | null = null;
 
+  /**
+   * Scroll animation class to be applied when element is within viewport.
+   *
+   * @private
+   */
+  private _effectClass?: string;
+
+  /**
+   * Scroll animation to be applied when elements are out of view.
+   *
+   * @private
+   */
+  private _exitEffectClass?: string;
+
   /**
    * Cleans observers upon update or exit, and creates new instances if needed.
    *
    * @param [options] The options.
    * @param [options.create] `true` to create the new intersection observer.
    */
   private _cleanAndCreateObservers({ create }: { create?: boolean } = {}) {
+    this._setAnimationClasses();
```

Comment:
Seems that this should be called in `updated()`, given it depends on `animation` property. Another option is simply maintaining a map of `animation` property and the CSS class being used.